### DISCUSSION TITLE
How to hide net maui tabbar correctly?

### DISCUSSION THREAD

**Discussion by Kanat9494** [dotnet/maui#14226] [Ideas]
How to hide net maui bottom tabs correctly?

To see it, create maui project, add two pages, name them FirstPage, SecondPage.

add shell bottom tabs, for example two tabs: 1st tab, 2nd tab, set datatemplate for them FirstPage.

Then create in FirstTab, a Button to navigate to the SecondPage.
I

https://user-images.githubusercontent.com/74175405/228209694-2a72e4d5-cf82-44a2-be6b-016b9c3f6333.mp4

n the SecondPage set "TabBarIsvisible=False", then you'll see that Shell.Tab's not disappearing immediately, how it was in Xamarin.Forms,

How can i fix that?

**Comment by Kanat9494** [comment]
In, xamarin forms and net maui tabbar disappears on the main(parent page), and only after that it navigates to childpage:

net maui:

https://user-images.githubusercontent.com/74175405/229085413-09b4a7e3-6f2d-4690-9b92-84e4620b92dc.mp4

xamarin forms:

https://user-images.githubusercontent.com/74175405/229085483-a2f650c4-7258-41e2-87d5-02cfdd2909bc.mp4

other apps:

https://user-images.githubusercontent.com/74175405/229085788-05809927-904b-4503-8b33-c8a45204e7a2.mp4




how can i achieve this behavior in net maui apps?

**Comment by rwecho** [comment]
@Kanat9494  Are you find any other solutions? thanks.

**Reply by Kanat9494** [reply]
No, I didn't

**Comment by Kanat9494** [comment]
No, i didn't

**Comment by Kanat9494** [comment]
what is it?:)

сб, 29 июл. 2023 г. в 06:50, Xyncgas ***@***.***>:

**Comment by Kiriakos41** [comment]
Any fix?

**Comment by RedEye-Developers** [comment]
any fix ?

**Comment by RedEye-Developers** [comment]
asked this Question in 2023, now 2025 still no answer.

**Comment by Kiriakos41** [comment]
@jfversluis

**Comment by eqpoqpe** [comment]
2025 😼 guys any solutions

**Reply by RedEye-Developers** [reply]
No lol

MAUI team not even see this messages for many yeas lol

**Reply by RedEye-Developers** [reply]
@jamesmontemagno  can we get answer for this ?, we are waiting before GTA 6 Trailer.
